packages/salesforcedx-lightning-lsp-common/src/index.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+export {
+  toResolvedPath,
+  isLWCRootDirectoryCreated,
+  relativePath,
+  pathStartsWith,
```

Comment:
no need to export.



In general, please review unnecessary exports from the top-level index files where knip doesn't handle the task for you.  good AI task